src/wasm-interpreter.h

```diff
@@ -2408,6 +2419,16 @@ template<typename GlobalManager, typename SubType> class ModuleInstanceBase {
     // Stack of <caught exception, caught catch's try label>
     SmallVector<std::pair<WasmException, Name>, 4> exceptionStack;
 
+  protected:
+    SubType* getMemoryInstance() {
+      // ctor-eval needs the imported memory from env
```

Comment:
this comment looks stale? or maybe I don't understand it.

Reply:
Yes this should've been removed. Fixed now.